src/scss/components/optional/_fullscreen-pop-over.scss

```diff
@@ -15,7 +15,7 @@
 $fullScreenPopOver-background        : $grey--offWhite;
 $fullScreenPopOver-action-background : $white;
 $fullScreenPopOver-padding           : spacing(x2);
-$fullScreenPopOver-border-color      : $grey--lighter;
+$fullScreenPopOver-border-color      : $grey--light;
```

Comment:
Should we be using `$color-border--interactive`?

Reply:
I'm not entirely sure tbh (as we don't have designs for these so this is just my best guess against what we have in zeplin).

We'll be switching these variables out for PIE variables soon (when I integrate that in the next couple of weeks) so I'll make sure I go through and update all the aliases where I can when that happens.